### PR TITLE
#3102: Create a bug report template

### PR DESCRIPTION
**Pull request by kwabenantim**
## Summary
This pull request adds a bug report template to GitHub Issues.

### CODE REVIEW

**Review by MauriceHendrix — APPROVED** [review]
looks good to me!

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**System**
+Please provide relevant details of your system, such as:
+ * OS version
+ * CMake version
+ * C++ compiler version
+ * Relevant Chaste dependency versions e.g. PETSc, Boost, Sundials etc.
```
Comment (mirams) [inline .github/ISSUE_TEMPLATE/bug_report.md:38]: You could ask for the output of TestChasteBuildInfo.hpp here.

**Review by mirams — CHANGES_REQUESTED** [review]
You could ask for the output of `TestChasteBuildInfo.hpp` which should give most of the info on dependencies and build type etc.

**Comment by kwabenantim** [comment]
> You could ask for the output of `TestChasteBuildInfo.hpp` which should give most of the info on dependencies and build type etc.

Thanks @mirams. I've added some information about getting the output from `TestChasteBuildInfo.hpp` .

**Comment by mirams** [comment]
Yeah, you don't want output on failure though, you want all the output when it passes as that will show the dependency versions

**Comment by kwabenantim** [comment]
> Yeah, you don't want output on failure though, you want all the output when it passes as that will show the dependency versions

Thanks. It's now `ctest -V -R TestChasteBuildInfo` without the ~~`--output-on-failure`~~.

**Review by mirams — APPROVED** [review]

**Pull request merged**